.github/workflows/pr-rebase-needed.yml

```diff
@@ -0,0 +1,20 @@
+name: "PR Needs Rebase"
+on:
+  # So that PRs touching the same files as the push are updated
```

Comment:
I don't understand this comment. The push even will only be for pushes to the repo itself, and most of us create branches on forks, most of the time.